hello-world-mi/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+displayName: Hello World in MI
+description: Hellow World project created using WSO2 Micro Integrator
```

Comment:
```suggestion
description: A Hello World project created using the WSO2 Micro Integrator
```

Reply:
Fixed in dbebb9a56cd9f001df2b9b4cbb98397a9ada986f